src/components/search/SearchResults.jsx

```diff
@@ -1,8 +1,99 @@
-const SearchResults = () => {
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import AgeIcon from '../../assets/icons/icon-age.svg';
+import LessonTypeIcon from '../../assets/icons/icon-lesson.svg';
+import ScheduleIcon from '../../assets/icons/icons-schedule.svg';
+
+const SearchResults = ({ classes, currentPage, totalPages, onPageChange }) => {
+  const navigate = useNavigate();
+  const handleClick = () => {
+    navigate('/register');
+  };
   return (
-    <>
-      <h1>SearchResults</h1>
-    </>
+    <div className="container m-auto ">
+      {classes.length > 0 ? (
+        <>
+          <div className="flex flex-col gap-6 mt-10 ">
+            {classes.map((classItem) => (
+              <div
+                key={classItem._id}
+                className="flex flex-row border h-[300px] rounded-lg  border-gray hover:shadow-lg cursor-pointer "
```

Comment:
maybe its better to add w-11/12 self-center and change height h-1/4

Reply:
still  working on styles.  Will remove hardcoded values while doing responsive